examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #55.                        del targetf
</pre><p>Unnecessary; same goes for the other occurrences</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>